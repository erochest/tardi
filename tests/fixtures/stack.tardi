1 2 swap
3 4 dup
5 6 7 rot
8 9 nip
10 11 over
12 13 drop
