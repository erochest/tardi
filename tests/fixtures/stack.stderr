2
1
3
4
4
6
7
5
9
10
11
10
12
